README.md

```diff
@@ -20,14 +19,18 @@ If you would like to contribute a pull request to help the project out the file
 If you have a performance optimization that you would like to suggest to us, please read our document over here:
 https://developer.jboss.org/wiki/PerformanceGatesForAcceptingPerformanceFixesInNarayana
 
-Requirements
-------------
+### Support Guarantees
+
+Although there are no formal guarantees to support this software the team and, we hope, the broader community will make every effort to resolve queries, fix bugs and add requested features as quickly as resources allow. Fixes and enhancements will be merged into the head branch and will not be backported to community releases.
```

Comment:
`head` -> `main` (possibly?)

"will not be backported" might not end up being true if it does get backported to (say) "6.0". Maybe we could say "Community fixes and enhancements can only be expected to be considered for merging onto the `main` branch." instead?

Reply:
Although head is the official github name, I am happy to use the term main but we must remember to change it if we ever rename the main branch, like we did when we moved from using the name master. Should I go ahead an rename it?

Reply:
I can see it's changed, and given what you changed it to works for me, I approved it. Thank you.

Reply:
@tomjenkinson ignore my previous comment - my last push already changed it to what you requested.